src/v/redpanda/application.cc

```diff
@@ -525,6 +526,12 @@ void application::initialize(
       .get();
     _cpu_profiler.invoke_on_all(&resources::cpu_profiler::start).get();
 
+    /*
+     * Disable the logger for protobuf; some interfaces don't allow a pluggable
+     * error collector.
+     */
+    google::protobuf::SetLogHandler(nullptr);
+
```

Comment:
I guess this is fine... is there any information we'll lose by setting this to `nullptr`?